spec/features/generic_file_new_form_spec.rb

```diff
@@ -1,88 +1,74 @@
 require 'spec_helper'
-describe 'generic file new', :type => :feature do
-  let(:user) { FactoryGirl.create :user }
-  let!(:community1) do
-    Collection.create( title: 'Community 1') do |c|
-      c.apply_depositor_metadata(user.user_key)
-      c.is_community = true
-      c.is_official = true
-    end
-  end
 
-  let!(:community2) do
-    Collection.create( title: 'Community 2') do |c|
-      c.apply_depositor_metadata(user.user_key)
-      c.is_community = true
-      c.is_official = true
-      c.is_admin_set = true
-    end
-  end
+describe 'generic file new', type: :feature do
+  let(:user) { FactoryGirl.create :user }
 
-  after :all do
-    cleanup_jetty
+  before do
+    sign_in user
+    visit '/files/new'
   end
 
   describe 'unfold agreement div', js: true do
-    before do
-      visit '/'
```

Comment:
this is silly, why go to the homepage and then go to the deposit form...lets just go to the deposit form